src/mpris.cpp

```diff
@@ -39,6 +42,11 @@ Mpris::~Mpris()
 {
 }
 
+QObject *Mpris::api_factory(QQmlEngine *, QJSEngine *)
+{
+    return new Mpris();
```

Comment:
Would it be better to make this also a c++ singleton and return the instance here or not desirable due to concurrency or other factors ? (would only make sense if this can be shared across processes)